content/news/026/index.md

```diff
@@ -32,6 +32,7 @@ Feel free to send PRs about your own projects!
 [Rust]: https://rust-lang.org
 [join]: https://github.com/rust-gamedev/wg#join-the-fun
 
+- [Rust GameDev Podcast](#rust-gamedev-podcast-6)
```

Comment:
@ozkriff I added the podcast section to the table of contents. If this is fine, then I'll go make another PR for the meetup, since I forgot to do it :+1: